### ISSUE TITLE
New logo for İmagefit

### ISSUE THREAD

**Issue by reallinfo** [vinyll/django-imagefit#26]
Hello I wanted to contribute to ImageFit. I designed a logo for it. If you like it, I'll send you files and pr.

Also, If you have a logo idea you want, I can design it.

![imagefit](https://user-images.githubusercontent.com/36298335/39098160-4eb3d16e-466f-11e8-8097-6e30bb8ace91.jpg)

**Comment by vinyll** [comment]
Cool initiative!

Il like the writing and the color. Less fond of the squarish _i_ in _f_ notion.
But that is **very** subjective.

Just keep in mind that we will take it whatever you decide as it does improve the project anyways! ;)

Here's a little participation to the brainstorm based on your suggestion:

![img_0868](https://user-images.githubusercontent.com/145172/39100810-1000c30c-4690-11e8-9e9e-63118cc26d9a.jpg)

The idea is a fitted image.
Fitted with the corners enclosing the image and the ruler slimming it.

In the bottom proposal I took the [icon of an image](https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Picture_font_awesome.svg/512px-Picture_font_awesome.svg.png) to represent the image, but it might make the logo too heavy?

**Comment by reallinfo** [comment]
No problem. Let's work as much as you like :) but sorry :/ I thought a lot, but I could not find an idea.

![if2](https://user-images.githubusercontent.com/36298335/39204507-a0c7719e-4800-11e8-9715-43b59a0c6825.jpg)

**Comment by reallinfo** [comment]
Yeah, i agree :)

**Comment by vinyll** [comment]
@reallinfo Do you want to create a pull request, create a folder at the root lie /icons and insert there the file as "logo.svg"?
I'm not sure how familiar you are with Git and GitHub. If you're not I can either assist you and make it on my end.

**Comment by reallinfo** [comment]
Of course! I will send you a pr. Is there a size you want?
  Also, I can update readme. 
This like;

![v ew](https://user-images.githubusercontent.com/36298335/39251538-3b6449b0-48ac-11e8-8e27-bc2ab6cd6cf2.jpg)

**Comment by reallinfo** [comment]
Okay, All is ready. Which should I add to the readme file? Rectangular or square?

**Comment by vinyll** [comment]
I would vote for the rectangular version in the readme if you have no preference

**Comment by reallinfo** [comment]
yeah, rectangular is better than it.  I sent pr, please check it. Do not hesitate to tell me if you have another request :)

**Comment by reallinfo** [comment]
Editable file is here.

https://drive.google.com/open?id=1F3CGoykKeGNcFoH68YqK4kWBJfQyLz97
